LibGAP.jl/src/conversion.jl

```diff
@@ -53,3 +55,5 @@ end
 from_gap_list_int16( x ) = from_gap_list_type( x, Int16 )
 from_gap_list_int32( x ) = from_gap_list_type( x, Int32 )
 from_gap_list_int64( x ) = from_gap_list_type( x, Int64 )
+from_gap_list_string( x ) = from_gap_list_type( x, String )
```

Comment:
... and this could perhaps become `from_gap(x,::Array{String})`, and so on? And then in fact, all of the above and including `from_gap_list_type` could be replaced by a single method `from_gap(x, ::Array{T} where T)`, no?

Not in this PR, mind you.